server_addon/substancepainter/client/ayon_substancepainter/plugins/publish/collect_context_data.py

```diff
@@ -1,17 +1,21 @@
 import pyblish.api
 
 from ayon_core.pipeline import registered_host
+from ayon_core.lib import get_version_from_path
 
 
-class CollectCurrentFile(pyblish.api.ContextPlugin):
-    """Inject the current working file into context"""
+class CollectContextData(pyblish.api.ContextPlugin):
+    """Collect current context publish"""
 
     order = pyblish.api.CollectorOrder - 0.49
-    label = "Current Workfile"
+    label = "Collect Context Data"
     hosts = ["substancepainter"]
 
     def process(self, context):
         host = registered_host()
         path = host.get_current_workfile()
         context.data["currentFile"] = path
         self.log.debug(f"Current workfile: {path}")
+        version = int(get_version_from_path(path))
+        context.data["version"] = version
+        self.log.debug(f"Current Version: {version}")
```

Comment:
Logic is basically the same as the `collect_workfile_version.py` file after.
```suggestion
```
Please revert the changes to this file.

Reply:
> Logic is basically the same as the `collect_workfile_version.py` file after.
> 
> Please revert the changes to this file.

I tried to search this particular python script in ayon_core but there is zero result. When I search similar scripts like `collect_version.py`, they are only lasted per host. Should we implement it in the general core addon?
![image](https://github.com/ynput/ayon-core/assets/64118225/852310fb-d461-4c75-a05f-be261ddff9fd)